packages/waku/src/lib/constants.ts

```diff
@@ -1,3 +1,4 @@
 export const EXTENSIONS = ['.js', '.ts', '.tsx', '.jsx', '.mjs', '.cjs'];
 export const SRC_MAIN = 'main';
 export const SRC_ENTRIES = 'entries';
+export const SRC_PAGES = 'pages';
```

Comment:
Technically, this is customizable, so not a constant like others. Let's move this to the plugin for now.